### PR TITLE
Python 3 compatibility

### PR DESCRIPTION
**Pull request by yvaucher**
Attempt to get python3 compatibility

Need to do the following pip installs:

```
pip install configparser
pip install future
```

### CODE REVIEW

File: scripts/migrate.py [hunk 1]
```diff
@@ -69,8 +72,9 @@ def copy_database(conn_parms):
         },
         'server': {
             'type': 'git',
-            'url': 'git://github.com/OpenUpgrade/OpenUpgrade.git',
-            'branch': '11.0',
+            #'url': 'git://github.com/OpenUpgrade/OpenUpgrade.git',
+            'url': 'src',
```
Comment (yvaucher) [inline scripts/migrate.py]: hack to remove

**Comment by yvaucher** [comment]
Rebased to have travis launched

File: .travis.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ language: python
 sudo: false
 
 python:
-  - "2.7"
+  - "3.5"
```
Comment (mreficent) [inline .travis.yml]: 3.6

Comment (mreficent) [reply .travis.yml]: ah, sorry, you were right. It's 3.5

**Review by StefanRijnhart — APPROVED** [review]
Thank you! Merging this, then continuing here: https://github.com/OCA/OpenUpgrade/pull/1119

**Pull request merged**